### ISSUE TITLE
driver name com.digitalocean.csi.dobs not found in the list of registered CSI drivers

### ISSUE THREAD

**Issue by Barvoj** [digitalocean/csi-digitalocean#86]
### What did you do? (required. The issue will be **closed** when not provided.)
* Created new Kubernetes cluster (v1.12.0) with one master, one worker and calico network using kubeadm
* Installed csi-digitalocean-v0.2.0 using instructions in readme.md
* Created PVC csi-pvc in section **3. Test and verify** of instructions
* Created POD my-csi-app
* Pod is in state ContainerCreating with this events (kubectl describe)
```
Events:
  Type     Reason                  Age                 From                     Message
  ----     ------                  ----                ----                     -------
  Normal   Scheduled               22m                 default-scheduler        Successfully assigned default/my-csi-app to worker1
  Normal   SuccessfulAttachVolume  21m                 attachdetach-controller  AttachVolume.Attach succeeded for volume "pvc-c3738d2cc47711e8"
  Warning  FailedMount             106s (x9 over 20m)  kubelet, worker1         Unable to mount volumes for pod "my-csi-app_default(e9139ae0-c477-11e8-917b-de5efd564cc5)": timeout expired waiting for volumes to attach or mount for pod "default"/"my-csi-app". list of unmounted volumes=[my-do-volume]. list of unattached volumes=[my-do-volume default-token-gbtz2]
  Warning  FailedMount             80s (x18 over 21m)  kubelet, worker1         MountVolume.MountDevice failed for volume "pvc-c3738d2cc47711e8" : driver name com.digitalocean.csi.dobs not found in the list of registered CSI drivers
```

### What did you expect to happen?
* Pod created and volume attached to it

### Configuration (**MUST** fill this out):

* system logs:

https://gist.github.com/Barvoj/6f15f342a1db88bd1717e4d92ec2915a

* manifests, such as pvc, deployments, etc.. you used to reproduce:

I created new cluster using this steps:
create 2 new digital ocean droplets (master and worker)

**Run on all nodes:**
```
apt-get update
apt-get upgrade

# Install docker as described https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce
apt-get install -y apt-transport-https ca-certificates curl software-properties-common
curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
apt-get update
apt-get install -y docker-ce

# Install kubeadm as described https://kubernetes.io/docs/setup/independent/install-kubeadm/

# Disable swap
swapoff -a

apt-get update && apt-get install -y apt-transport-https curl
curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
cat <<EOF >/etc/apt/sources.list.d/kubernetes.list
deb http://apt.kubernetes.io/ kubernetes-xenial main
EOF
apt-get update
apt-get install -y kubelet kubeadm kubectl
apt-mark hold kubelet kubeadm kubectl
```

**Run on master:**
```
# Create cluster as described https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/

kubeadm init --pod-network-cidr=10.244.0.0/16 --apiserver-advertise-address=68.183.36.31

kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/rbac.yaml
kubectl apply -f https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/hosted/canal/canal.yaml
```

**Run on worker:**
kubeadm join from output of kubeadm init

**Make kubelet privileged on all nodes**
Add --allow-privileged=true in /etc/systemd/system/kubelet.service.d/10-kubeadm.conf for KUBELET_KUBECONFIG_ARGS and then systemctl restart kubelet

**Instal CSI**
I followed https://github.com/digitalocean/csi-digitalocean#1-create-a-secret-with-your-digitalocean-api-access-token

* CSI Version: v0.2.0
* Kubernetes Version: v1.12.0
* Cloud provider/framework version, if applicable (such as Rancher): kubeadm v1.12.0

**Comment by fatih** [comment]
Thanks for the feedback. Seems like CSI spec is broken in v1.12.0. The way of registration of drivers have changed. We already have `NodeGetID()` and `NodeGetInfo()`. I'm looking into now and will post more about it.

**Comment by fatih** [comment]
Here are the steps needed to make the changes for `v1.12.0`: https://kubernetes-csi.github.io/docs/Setup.html

**Comment by fatih** [comment]
This is now fixed with our latest `v0.3.0` release. Here is the changelog for it: https://github.com/digitalocean/csi-digitalocean/blob/master/CHANGELOG.md#v030---20181029

And here is the deployment file: https://github.com/digitalocean/csi-digitalocean/blob/master/deploy/kubernetes/releases/csi-digitalocean-v0.3.0.yaml

**Comment by dperetti** [comment]
This is happening to me right now in a DO cluster 🤔
